workflows/java/sdk/order-processor/src/main/java/io/dapr/quickstarts/workflows/WorkflowConsoleApp.java

```diff
@@ -97,13 +97,13 @@ private static void executeWorkflow(DaprWorkflowClient workflowClient, Inventory
           Duration.ofSeconds(30),
           true);
       if (workflowStatus != null) {
-        System.out.printf("workflow instance completed, out is: %s%n",
+        System.out.printf("Workflow instance completed, out is: %s%n",

```

Comment:
Changing this has caused the test to fail since in the Readme it is looking for lowercase "w". Can you update the readme file as well to look for the upper case "W"?
<img width="1057" alt="image" src="https://github.com/user-attachments/assets/b6e7f7ef-671b-4ccd-b0df-e38cf943a333" />